app/views/items/edit.html.erb

```diff
@@ -2,4 +2,51 @@
 
 <%= link_to "Back to list", list_path(@list), class: "pull-right" %>
 
-<%= render "form", url: item_path(list_id: params[:list_id]) %>
+<%= form_for @item, url: item_path(list_id: params[:list_id]) do |f| %>
+  <%= render "shared/alert", invalid_object: @item, name: "item" %>
+
+  <%= f.hidden_field :user_id, value: current_user.id %>
+  <%= f.hidden_field :purchased, value: false %>
+
+  <div class="form-group" id="new-item">
+    <%= f.label :name do %>
+      Add a new item
+    <% end %>
+    <%= f.text_field :name, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :quantity %>
+    <%= f.number_field :quantity, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :quantity_name %>
+    <%= f.text_field :quantity_name, class: "form-control" %>
+    <small class="help-block text-muted">
+      This is meant to be used in conjunction with quantity. For example "1 bag"
+      or "12 ounces".
+    </small>
+  </div>
+
+  <%= f.submit "Submit", class: "btn btn-primary" %>
+<% end %>
+
+<script>
```

Comment:
Yeah. It'll be nice to move away from these views and into React so we can just throw our JS in when we need it.